dns/bind/src/opnsense/scripts/OPNsense/Bind/dnsbl.sh

```diff
@@ -137,7 +138,14 @@ install() {
 	rm -rf ${WORKDIR}
 }
 
-for CAT in $(echo ${1} | tr ',' ' '); do
+if [ $# -eq 0 ] && [ -s "$DESTDIR/dnsbl.list" ]; then
+    DNSBLLIST=$(cat ${DESTDIR}/dnsbl.list)
+else
+    DNSBLLIST=$1
+fi
+echo "$DNSBLLIST" > ${DESTDIR}/dnsbl.list
```

Comment:
Will there be longtime happiness with this side effect caching? when is the script *not* called with the argument so it needs magical recovery? :)

Reply:
By cron .. :/